style.css

```diff
@@ -435,8 +468,161 @@ footer {
   margin-top: 15px;
 }
 
+.contact {
+  align-items: center;
+}
+
 .email {
   text-align: center;
   margin-left: 20px;
   color: #42526e;
 }
+
+.scroll_hide_desk {
+  display: none;
+}
+
+@media (min-width: 1024px) {
+  .background {
+    background-image: url("img/background_desktop.svg");
+    background-size: cover;
+    background-position-x: 50%;
+    background-position-y: 60%;
+  }
+
+  .scroll_hide_desk {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .hide_mobile {
+    display: flex;
+    justify-content: space-between;
+    margin: auto;
+    padding: 0 0;
+  }
+
+  .hide_mobile li {
+    margin-left: 2rem;
+    margin-top: 1rem;
+  }
+
+  .greeting {
+    padding-top: 150px;
+  }
+
+  .connect {
+    padding: 0%;
+    margin-bottom: 1rem;
+  }
+
+  .presentation + .greeting {
+    margin-top: -150px;
+  }
+
+  .presentation {
+    margin: 0 0;
+    padding-top: 10px;
+    font-size: 15px;
+  }
+
+  .indicator {
+    display: none;
+  }
+
+  .scroll_hide_mobile {
+    margin-bottom: 150px;
+    margin-top: 50px;
+  }
+
+  .wrapper {
+    grid-template-columns: repeat(3, 0.1fr);
+    grid-gap: 1rem 1rem;
+    grid-auto-rows: minmax(40px, auto);
+    justify-content: center;
+  }
+
+  #aboutme {
+    background-image: url("img/background_aboutme.svg");
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .resume-tech {
+    display: flex;
+  }
+
+  .coding-languages,
```

Comment:
- [ ] Ensure that padding/margin on the left and right on all sections are equal.
- 'My recent works' has perfect padding/margin on the sides. Do the same for the technology section and the contact section.
![Screenshot from 2021-08-20 11-57-14](https://user-images.githubusercontent.com/65501535/130208844-0f2fe3d9-2ddb-41cf-af9c-8cbe4edf7a5c.png)